Assets/__Scripts/Beatmap/Enums/ExecutionTime.cs

```diff
@@ -0,0 +1,8 @@
+namespace Beatmap.Enums
+{
+    public enum ExecutionTime
+    {
+        Early,
+        Late
+    }
```

Comment:
Is this enum used?

Reply:
should be in rotation event, tho many in enums are just placeholder or for readability purpose